[2024-05-06T07:50:31.165+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-06T07:50:31.320+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.upload_file_orders_to_capstone scheduled__2024-05-05T00:00:00+00:00 [queued]>
[2024-05-06T07:50:31.576+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.upload_file_orders_to_capstone scheduled__2024-05-05T00:00:00+00:00 [queued]>
[2024-05-06T07:50:31.576+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-06T07:50:32.018+0000] {taskinstance.py:2327} INFO - Executing <Task(GCSToGCSOperator): upload_file_orders_to_capstone> on 2024-05-05 00:00:00+00:00
[2024-05-06T07:50:32.246+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=230) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-06T07:50:32.253+0000] {standard_task_runner.py:63} INFO - Started process 303 to run task
[2024-05-06T07:50:32.260+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl', 'upload_file_orders_to_capstone', 'scheduled__2024-05-05T00:00:00+00:00', '--job-id', '285', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpmo5_izbg']
[2024-05-06T07:50:32.290+0000] {standard_task_runner.py:91} INFO - Job 285: Subtask upload_file_orders_to_capstone
[2024-05-06T07:50:32.681+0000] {task_command.py:426} INFO - Running <TaskInstance: etl.upload_file_orders_to_capstone scheduled__2024-05-05T00:00:00+00:00 [running]> on host c8cbe72b3ec5
[2024-05-06T07:50:35.430+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='upload_file_orders_to_capstone' AIRFLOW_CTX_EXECUTION_DATE='2024-05-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-05T00:00:00+00:00'
[2024-05-06T07:50:35.431+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-06T07:50:35.449+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-05-06T07:50:35.457+0000] {base.py:84} INFO - Using connection ID 'my_gcp_conn' for task execution.
[2024-05-06T07:50:37.209+0000] {gcs_to_gcs.py:535} INFO - Executing copy of gs://raw_data_projectcapstone/olist_orders_dataset.csv to gs://storage-capstone/olist_orders_dataset.csv
[2024-05-06T07:50:37.300+0000] {gcs.py:256} INFO - Total Bytes: 17654914 | Bytes Written: 17654914
[2024-05-06T07:50:37.301+0000] {gcs.py:264} INFO - Object olist_orders_dataset.csv in bucket raw_data_projectcapstone rewritten to object olist_orders_dataset.csv in bucket storage-capstone
[2024-05-06T07:50:37.301+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-06T07:50:37.337+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl, task_id=upload_file_orders_to_capstone, execution_date=20240505T000000, start_date=20240506T075031, end_date=20240506T075037
[2024-05-06T07:50:38.153+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-06T07:50:43.352+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-06T07:50:43.362+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
